docs/user-guide/cli-using-initializing-team-configuration.md

```diff
@@ -89,11 +120,17 @@ If you are using the API Mediation Layer, set up the `zowe.config.json` file to
 
     A default profile is added to the configuration file in the `.zowe` home directory.
 
-**Note:** To add a profile to a configuration file in the current working directory, enter the base command without the `--global-config` option: `zowe config init`.
+:::note
+
+To add a profile to a configuration file in the current working directory, enter the base command without the `--global-config` option: `zowe config init`.
+
+:::
+
+## Using Certificates
 
-**Using Certificates:**
+**[what does this apply to? api ml? everything in this article?]**
```

Comment:
@adam-wolfe @zFernand0 - What does this section apply to? Not sure when/where the user is supposed to issue this command and for what use case.

Reply:
Doesn't z/OSMF also support certificate authentication? Or would that overcomplicate this section?

Reply:
@zFernand0 after MUCH deliberation with you in the office, we have figured this out and edited this content as needed.